vars/transportRequestUploadRFC.groovy

```diff
@@ -0,0 +1,17 @@
+import com.sap.piper.BuildTool
+import groovy.transform.Field
+import static com.sap.piper.Prerequisites.checkScript
+import com.sap.piper.DownloadCacheUtils
+
+@Field String STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/transportRequestUploadRFC.yaml'
+
+void call(Map parameters = [:]) {
+    final script = checkScript(this, parameters) ?: this
+    parameters = DownloadCacheUtils.injectDownloadCacheInParameters(script, parameters, BuildTool.MTA)
```

Comment:
What does this do?

Reply:
As far as I understood, `DownloadCacheUtils.injectDownloadCacheInParameters ` induces, beside others, the context config params, like `dockerImage`.  I always wonder about the nameing.

Reply:
I see. And why is there a reference to the `BuildTool.MTA`?

Reply:
well, i guess because this method does 1000 things, e.g. setting up some keys for thee build tools. with choosing MTA you dont miss a parameter (at the moment).